applications/metadata.js

```diff
@@ -43,6 +43,15 @@ var metadata = { "applications": [
          "jsfile": "domain-navigator/dom_nav.js",
          "icon": "domain-navigator/dom_nav_icon_i.png",
          "caption": "DOM NAV"
-      }
+     },
+     {
+        "isActive": true,
+        "directory": "vr-grabscale",
+        "name": "VR Grabscale",
+        "description": "This tool allows you to scale objects by grabbing them in VR.",
+        "jsfile": "vr-grabscale/VRBuildGrabScale.js",
+        "icon": "vr-grabscale/logo.png",
+        "caption": "VR SCALE"
+     }
    ]
```

Comment:
Indentation is messed up. This isn't particular to this PR so I've opened an issue: https://github.com/kasenvr/community-apps/issues/7

Reply:
Metadata is regenerated, so we would have to fix that in the generator since when someone opens a PR they'll be attempting to set it back that way anyway.